packages/cli/src/formatters/__tests__/__fixtures__/oas3-schema-errors.json

```diff
@@ -51,7 +51,7 @@
   },
   {
     "code": "oas3-schema",
-    "message": "should have required property '$ref'",
+    "message": "should have required property '$ref' and handle special strings like \", <, > or ]]> well",
```

Comment:
this is a serialized output of `spectral.run`, thus it might be better to leave it as is and simply add another test case to `packages/cli/src/formatters/__tests__/junit.test.ts`